### PR TITLE
[v6r14] Fixes in SE machinery

### PR DESCRIPTION
**Pull request by atsareg**

### CODE REVIEW

File: Resources/Storage/StorageFactory.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def getStorages( self, storageName, pluginList = None ):
     if pluginList is None:
       pluginList = []
     if not self.vo:
-      return S_ERROR( 'Mandatory vo parameter is not defined' )
+      gLogger.warn( 'No VO information available' )
```
Comment (chaen) [inline Resources/Storage/StorageFactory.py]: why isn't it mandatory anymore ?

Comment (atsareg) [inline Resources/Storage/StorageFactory.py]: It is used if it is defined ( for some possible VO specific parameters ). If not defined, nothing specific. For example, like that it allows usage by dirac_admin group which is not hooked up on any VO. We have to think more about VO-group-user relations still. However, for the moment this is more harm than help if the VO parameters is mandatory

Comment (chaen) [inline Resources/Storage/StorageFactory.py]: I see... For us it does not change anything anyway, however you are right, one should think more in depth about the multi VO paradigm. But maybe, one should do it before doing small patches here and there ?

**Comment by chaen** [comment]
Review OK

File: Resources/Storage/StorageFactory.py (hunk 1 above)
Comment (andresailer) [inline Resources/Storage/StorageFactory.py]: Why is this still mandatory in v6r13? Changing this also allows us to download sandboxes as diracAdmin.

**Comment by atsareg** [comment]
Indeed, this PR should got o v6r13 patch. Will do.

**Comment by atsareg** [comment]
Redirected to rel-v6r13

**Pull request closed without merging**